### DISCUSSION TITLE
Commercial Data

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/california-data#4] [General]
Use this discussion board to ask questions related to commercial data

**Comment by melissamonk-NOAA** [comment]
BJL-NWFSC: Canary landings data in PacFIN come from fewer than 3 vessels for a few years. For the assessment tables and .dat files, the landings are added to discards to come up with total commercial mortality. Do you foresee confidentiality concerns with showing the commercial catches (landings and discards)? Would showing commercial landings even be a concern - because a single number wouldn’t necessarily convey who makes up those landings.

**Reply by melissamonk-NOAA** [reply]
@Ross-Cooper-CDFW I added your answer from the GDrive here.
A: RC - CDFW: Hi Brian, would you be able to send me the years in question so I can check it out on my end as well? CA’s commercial fisheries data confidentiality requirements are outlined in [FGC 8022](https://leginfo.legislature.ca.gov/faces/codes_displaySection.xhtml?lawCode=FGC&sectionNum=8022.), and the “Rule of Three” is fairly strict in that records summarized for public disclosure must be the aggregate of at least three distinct fishers, three distinct vessels, and three distinct fish businesses. This three-unit threshold is based on instances where in subsets of certain fisheries, such as records for a certain species, from a certain region, in a certain period, there are exactly two participants that are aware of each other and would be able to calculate the other participants' data.  

I don’t think splitting landings & discards or combining them into total commercial mortality will have any real bearing on confidentiality, because the data confidentiality is ultimately determined up front according to the number of distinct fishers / vessels / fish businesses. With that said, I am brand new to stock assessments and am not sure what resolution we are referencing here (CA region vs state vs entire west coast, etc.) and broader resolution would likely have fewer data redactions due to confidentiality. This is a common hurdle for us to clear with our data publications, and I am happy to discuss further details and brainstorm solutions if you want!

**Reply by brianlangseth-NOAA** [reply]
I provided Ross the years, and communicated that discards are calculated from WCGOP data. Thus the total values used in the .dat files and in the report for PacFIN years are combined across the PacFIN landings data and the WCGOP discards data.

**Reply by Ross-Cooper-CDFW** [reply]
Hi Brian and Melissa,
I was discussing this with by boss last Friday and he is looping in a few more people today, once I have a more concrete answer I will report back.

**Reply by iantaylor-NOAA** [reply]
@brianlangseth-NOAA, while CDFW is discussing this, here are a few thoughts from the modeling side of things.

Table 6 of the 2015 Canary assessment report: https://www.pcouncil.org/documents/2016/05/the-status-of-canary-rockfish-sebastes-pinniger-in-the-california-current-in-2015-march-2016.pdf/#page=84 shows some fleet/year combinations with less than 1 ton of catch and I presume that the fewer-than-3-vessel cases you're seeing are similarly small. On the one hand, excluding that catch from the model entirely (fixing the catch at 0), or setting all catch associated with than 3 vessels to exactly 1.0 tons, would surely have almost zero impact on the model estimates. Likewise combining trawl and non-trawl catch for the years in question would again surely have zero impact on the overall results. The biggest problem I can see with both of those approaches would be the potential impact on estimates of the distribution of catch among fleets used in forecasts. However, you could confirm that any recent average catch estimates used in forecasting or for reference point calculations had only negligible impact in which case any of those modifications would seem reasonable as long as the change was adequately documented in the assessment report and table captions.

**Reply by Ross-Cooper-CDFW** [reply]
@iantaylor-NOAA @brianlangseth-NOAA Those are good points Ian. I am holding off on concrete statements for some aspects of this until I get confirmation back from my CDFW contacts, but I believe you can still use that data for the data modeling regardless of confidentiality. The confidential-redactions mostly come into play when displaying / publishing certain data in a public-facing format, but for modeling / analyses they can still be used (provided potentially confidential data isn't able to be deduced from total value calculations). Once I get confirmation, I would be happy to jump on a call or cover this on Wednesday (provided I have an answer by then).

**Reply by iantaylor-NOAA** [reply]
@Ross-Cooper-CDFW, thanks for the follow up. We are required to publish the model input files which show the catch by fleet and year so it's not possible to use values in the aggregated model inputs that are not publishable. However, it is possible to have a parallel model which includes confidential values to do an internal comparison to confirm that the final published model is not impacted by any changes required to preserve confidentiality of the inputs.

**Reply by Ross-Cooper-CDFW** [reply]
@iantaylor-NOAA @brianlangseth-NOAA That is good to know, and I think the parallel model idea is good to confirm since the data being redacted usually tends to be relatively negligible in the grand scheme of things. Do we have a contact with the WCGOP to confirm how they define their confidentiality restrictions? I believe it is just based on 3 distinct vessel IDs whereas CA does 3 distinct vessel IDs, fisher IDs, and fish businesses, but obviously fish businesses don't come into play when considering observer data anyways. I would be happy to setup a call with us (and likely some others) to chat about this to make sure we are all on the same page, both in terms of making sure confidentiality rules are adhered to as well as not mistakenly redacting data that is actually ok to use.

**Comment by chantelwetzel-noaa** [comment]
@Ross-Cooper-CDFW WCGOP applies their confidentiality rule based on 3 distinct vessel IDs.

**Comment by brianlangseth-NOAA** [comment]
@Ross-Cooper-CDFW I support your idea for a call. Easiest way to get on the same page.

**Comment by brianlangseth-NOAA** [comment]
Met with CDFW (@Ross-Cooper-CDFW, @MelMandrupCDFW, Paulo Serpa, and John Budrick) and NOAA staff (@brianlangseth-NOAA, and Kate Richerson) to discuss. 

The short answer is that confidential landings data aggregated with non-confidential discard data should be ok. 

For some species that are managed on a state by state basis, it may be relatively easy to know discards from the GEMM report and therefore back-calculate landings, however for Canary, this information is not available from the GEMM so should be ok. The Council has a reasonableness guideline that although something could be back-calculated after a lot of work, if it cannot reasonably be, then that would preserve confidentiality. Of course, the definition of reasonable is not well defined. 

Action items for the future to confirm is 
-(for @brianlangseth-NOAA) to work with the observer program to determine number of fishers/vessels contributing to discard estimates, and then provide total mortality values to CDFW to ensure confidentiality is maintained - provision to the state is done anyway as confirmation of values going into the model.

**Reply by brianlangseth-NOAA** [reply]
Related to the above, CDFW interprets confidentiality to apply to the domain of what is being reported. Therefore, there must be 3 fisher/vessels/dealers that caught canary on a gear group. It does not apply to 3 fisher/vessels/dealers that fished in the gear group that could have caught canary.
